qt/python/mantidqt/widgets/test/test_fitpropertybrowser.py

```diff
@@ -32,7 +31,10 @@ def tearDown(self):
         AnalysisDataService.clear()
 
     def test_initialization_does_not_raise(self):
-        assertRaisesNothing(self, self._create_widget)
+        try:
```

Comment:
The correct way of testing this is:
```python
self.assertIsNone(self._createWidget())
```
Or alternatively
```python
self._createWidget()
```

All unit tests assume that exceptions are failures so the try-expect is redundant and will likely set off future static analysis about the bare except

Reply:
But create_widget does return an object, so `assertIsNone` would always fail surely. To best honest this test is redundant as you say, as the method is called in every subsequent test, so we'll always know about it. I've removed it.